community/values/src/test/java/org/neo4j/values/storable/TimeValueTest.java

```diff
@@ -246,19 +241,18 @@ private DateTimeException assertCannotParse( String text )
         throw new AssertionError( text );
     }
 
-    private static NanoOfDayAndOffset write( TimeValue value )
+    private static OffsetTime write( TimeValue value )
     {
-        NanoOfDayAndOffset result = new NanoOfDayAndOffset();
+        AtomicReference<OffsetTime> result = new AtomicReference<>();
         value.writeTo( new ThrowingValueWriter.AssertOnly()
         {
             @Override
-            public void writeTime( long nanosOfDayUTC, int offsetSeconds )
+            public void writeTime( OffsetTime offsetTime )
             {
-                result.nanosOfDay = nanosOfDayUTC;
-                result.offsetSeconds = offsetSeconds;
+                result.set( offsetTime );
             }
         } );
-        return result;
+        return result.get();
     }
 
     private static class NanoOfDayAndOffset
```

Comment:
This class us not used any more.

Reply:
Addressed in https://github.com/neo4j/neo4j/commit/68ea67b3082b2fe3e070a13a7ec8cae6ae5e786d